.gitlab/pipeline/test.yml

```diff
@@ -523,3 +523,4 @@ test-syscalls:
     - if [[ "$CI_JOB_STATUS" == "failed" ]]; then
         printf "The x86_64 syscalls used by the worker binaries have changed. Please review if this is expected and update polkadot/scripts/list-syscalls/*-worker-syscalls as needed.\n";
       fi
+  allow_failure: true # TODO: remove this once we have an idea how often the syscall lists will change
```

Comment:
Was probably added to not confuse devs who would be blocked and not know what to do.
But the syscalls should change rarely and mostly only when updating the worker code.

It can stay not required for now until we have the syscall blocking in place.